apps/frontend/src/app/Components/FieldDisplay.tsx

```diff
@@ -121,25 +124,49 @@ export function NodeFieldDisplay({
   const { multi } = node.info
 
   const multiDisplay = multi && <span>{multi}&#215;</span>
+  const nodeValue = node.value
   let fieldVal = false as ReactNode
-  if (oldValue !== undefined) {
-    const diff = node.value - oldValue
-    const pctDiff = valueString(Math.abs(diff / oldValue), '%', node.info.fixed)
+  if (compareValue !== undefined) {
+    const diff = nodeValue - compareValue
+    const pctDiff = valueString(diff / compareValue, '%', node.info.fixed)
     fieldVal = (
       <>
-        <span>{valueString(oldValue, node.info.unit, node.info.fixed)}</span>
+        <span>{valueString(nodeValue, node.info.unit, node.info.fixed)}</span>
         {Math.abs(diff) > 0.0001 && (
-          <>
-            <ColorText color={diff > 0 ? 'success' : 'error'}>
-              {diff > 0 ? '+' : ''}
-              {valueString(diff, node.info.unit, node.info.fixed)}
+          <Tooltip
+            title={
+              <Typography>
+                Compare to{' '}
+                <ColorText color={diff > 0 ? 'success' : 'error'}>
```

Comment:
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/7ab3541b-ac84-4a6d-b963-7bd2408d93de)
This coloring is a little confusing honestly. Might be better without any coloring?

Like it doesn't seem right that the lower number in the tooltip is green for some reason, as if indicating it is better. Im not sure if there would be similar confusion when its flipped